src/AC/AC2/Thermocouples.cpp

```diff
@@ -116,4 +116,9 @@ namespace Thermocouples {
         return tcUpdatePeriod;
     }
 
+    bool abortAll() {
```

Comment:
you should reference this ip's in the HAL or a config file so you don't have to change them everytime you change an ip